packages/node_modules/@ciscospark/spark-core/src/spark-core.js

```diff
@@ -396,6 +365,37 @@ const SparkCore = AmpState.extend({
     this.sessionId = sessionId;
   },
 
+  /**
+   *
+   * Check if access token is correctly formated and correct if it's not
+   * Warn user if token string has errors in it
+   * @param {string} token
+   * @returns {string}
+   */
+  bearerValidator(token) {
+    if (token.includes('Bearer') && token.split(' ').length - 1 === 0) {
+      console.warn(
```

Comment:
Instead of `console`, use the `logger`.

Reply:
`this.logger` is not in scope because bearerValidator gets called during the constructor and it seems that the logger gets added to the initialized variable/object somewhere else (at least not in the `spark-core` file)